android-json-form-wizard/src/main/java/com/vijay/jsonwizard/fragments/JsonWizardFormFragment.java

```diff
@@ -317,7 +348,7 @@ private void checkIfStepIsBlank(JSONObject formStep) {
                 for (int i = 0; i < fields.length(); i++) {
                     JSONObject field = fields.getJSONObject(i);
                     if (field.has(JsonFormConstants.TYPE) && !JsonFormConstants.HIDDEN.equals(field.getString(JsonFormConstants.TYPE))) {
-                        boolean isVisible = field.optBoolean(JsonFormConstants.IS_VISIBLE, true);
+                        boolean isVisible = field.optBoolean(JsonFormConstants.IS_VISIBLE, false);
```

Comment:
what is the reason of changing the default from true

Reply:
my mistake, let me revert it